AutoGrade/urls.py

```diff
@@ -4,16 +4,17 @@
 
 urlpatterns = [
     url(r'^$', views.home, name='home'),
-    
+
     url(r'^login/$', auth_views.login, {'template_name': 'login.html'}, name='login'),
     url(r'^logout/$', auth_views.logout, {'next_page': 'login'}, name='logout'),
     url(r'^signup/$', views.signup, name='signup'),
     url(r'^api/(?P<action>[0-9a-zA-Z_]+)$', views.api, name='api'),
-    
+
     url(r'^assignment_report/(?P<assignment_id>[0-9a-zA-Z_]+)$', views.assignment_report, name='assignment_report'),
+    url(r'^assignment_aggregate_report/(?P<assignment_id>[0-9a-zA-Z_]+)$', views.assignment_aggregate_report, name='assignment_aggregate_report'),
     url(r'^moss_submit/(?P<assignment_id>[0-9a-zA-Z_]+)$', views.moss_submit, name='moss_submit'),
     url(r'^moss_view/(?P<assignment_id>[0-9a-zA-Z_]+)$', views.moss_view, name='moss_view'),
-    
```

Comment:
@recluze Which editor do you use? which find out these spaces/tabs? or you run autopep8?

Reply:
It's `atom`. I think I might have a PEP plugin installed for this (not sure).